routes/users.js

```diff
@@ -98,7 +98,7 @@ router
     //user = await getUserByUserName(username);
     try {
       user = await getUserByUserName(username);
-      if (user)
+      if (!Object.keys(user).length === 0){}
```

Comment:
What's going on here? Why is there nothing in the body of the if statement?

Reply:
It must be something wrong. There is an error report in the if body. You can check the route file at line 102.